packages/govuk-frontend-review/src/app.mjs

```diff
@@ -275,8 +275,10 @@ export default async () => {
     }
   )
 
-  // Full page example views
-  routes.fullPageExamples(app)
+  /**
+   * Full page example routes
+   */
+  app.use('/full-page-examples', routes.fullPageExamples)
```

Comment:
Like how this makes the start of the path obvious and denotes it's responsible for its own thing 😍